brainpy/_src/math/compat_pytorch.py

```diff
@@ -68,4 +68,21 @@ def flatten(input: Union[jax.Array, Array],
   return jnp.reshape(input, new_shape)
 
 
+def unsqueeze(input: Union[jax.Array, Array], dim: int) -> jax.Array:
+    """Returns a new tensor with a dimension of size one inserted at the specified position.
+  The returned tensor shares the same underlying data with this tensor.
+  A dim value within the range [-input.dim() - 1, input.dim() + 1) can be used.
+  Negative dim will correspond to unsqueeze() applied at dim = dim + input.dim() + 1.
+  Parameters
+  ----------
+  input: Array
+    The input Array
+  dim: int
+    The index at which to insert the singleton dimension
 
+  Returns
+  -------
+  out: Array
+  """
+    input = _as_jax_array_(input)
+    return jnp.expand_dims(input, dim)
```

Comment:
It's better to return a ``brainpy.math.Array``. This is because brainpy Array has more good properties compared with ``jax.Array``. 

We can rewrite this line as 

```
return Array(jnp.expand_dims(input, dim))

# or 

return _return(jnp.expand_dims(input, dim))

```